### PR TITLE
Select search box contents on focus (Attempt #2)

### PR DESCRIPTION
**Pull request by jpouellet**

### CODE REVIEW

File: qubesmanager/main.py [hunk 1]
```diff
@@ -119,6 +119,23 @@ def process_IN_CREATE(self, event):
...
+
+    def focusInEvent(self, e):
+        super(SearchBox, self).focusInEvent(e)
+        self.selectAll()
+        self.focusing = True
+
+    def mousePressEvent(self, e):
+        super(SearchBox, self).mousePressEvent(e)
+        if self.focusing:
+            self.selectAll()
```
Comment (jpouellet) [inline qubesmanager/main.py]: This one is required because the mousePressEvent otherwise deselects the text

**Comment by marmarek** [comment]
While I see the downside of not having preview in QtDesigner, this one looks better. Maybe it's possible to set some minimal height of that layout, to at least have some hint how it looks like?

**Comment by jpouellet** [comment]
> Maybe it's possible to set some minimal height of that layout, to at least have some hint how it looks like?

There is already the "Search:" label which accomplishes that:
qt-des-search
which I was actually going to remove in favor of [setPlaceholderText](https://doc.qt.io/qt-4.8/qlineedit.html#placeholderText-prop)("Search..."), but now perhaps not.

**Comment by marmarek** [comment]
Ok, so this looks fine.

**Pull request merged**